core/primitives/src/utils.rs

```diff
@@ -409,17 +408,7 @@ pub fn to_timestamp(time: DateTime<chrono::Utc>) -> u64 {
 
 /// Compute number of seats per shard for given total number of seats and number of shards.
 pub fn get_num_seats_per_shard(num_shards: NumShards, num_seats: NumSeats) -> Vec<NumSeats> {
-    (0..num_shards)
-        .map(|i| {
-            let remainder = num_seats % num_shards;
-            let num = if i < remainder as u64 {
-                num_seats / num_shards + 1
-            } else {
-                num_seats / num_shards
-            };
-            max(num, 1)
-        })
-        .collect()
+    vec![num_seats; num_shards as usize]
```

Comment:
so we'll assign the same number of seats for each shard? 

This changes the meaning of this function - and we should at least update the comment.

Reply:
Originally this function splits the validators among shards, so each validator is only assigned to one shard. I don't think this is what we want right now, especially for localnet, since in the current stage all validators track all shards. I'll update the comments.